### DISCUSSION TITLE
Apple TV live video notification feed without "someone rang the door bell" on motion

### DISCUSSION THREAD

**Discussion by ashishgururani** [dgreif/ring#916] [Q&A]
I could have sworn last year I used to get live video popup on Apple TV when there was motion without the notification "someone rang the door". I moved the homebridge instance to another device and some settings go reset. Since then I see option on the settings page to do this but seems like now the only option is that on motion, I get a notification on all homekit devices that "someone rang the doorbell" for the motion events, before it shows up on Apple TV video popup. Am I mistaken or was this feature never there?

**Comment by dgreif** [comment]
My understanding is that the latest tvOS version has an option to allow streaming on motion directly, rather than using the simulate setting in homebridge-ring.  Would you mind digging around the UI in tvOS to confirm if that is true?
